### DISCUSSION TITLE
TLDR-Tooltipps reset to Regular not working

### DISCUSSION THREAD

**Discussion by BobMahoo** [Lailloken/Exile-UI#471] [Q&A] [answered]
Hi i play poe at Vulkan, 4K - 50% resolution scaling Win 10, borderrless.
Lailloken works fine for me.
 TLDR-Tooltip means "Right mouse click" for reset the colour. Numbers works fine, r-mouse click dont do anything.
Maybe you can pull the reset to a numer key like "0" for resetting the colou or push the same number twice.

Best regards

**Comment by Lailloken** [comment, marked as answer]
There's a misunderstanding: The help-tooltip says there are two buttons for each pattern that you can left- or right-click
![image](https://github.com/user-attachments/assets/1bbac183-62dc-42c6-8e00-97885b327e2a)

Right-clicking resets the font/background color to the pattern's default.

While using TLDR-Tooltips in game, you press number-key X to apply pattern X, or you press SPACE to apply the "regular" pattern.

**Comment by BobMahoo** [comment]
Thanks for fast replay.

The reset with space are very helpful for that. Thanks. 
I think it should be written within the "help-tooltip"!

Your "help-tooltips" are very good i think. They help alot to config the app to your prefs..! 
I missunderstood that right click was meaning for the config of colour management. ^^

Thanks for that.
